### PR TITLE
Added IronSocket OVPN files

### PR DESCRIPTION
**Pull request by brossow**
Added IronSocket OVPN files from https://ironsocket.com/network#ovpn (current as of 2018-05-11)

### CODE REVIEW

File: openvpn/ironsocket/Argentina.ovpn [hunk 1]
```diff
@@ -0,0 +1,196 @@
+setenv FORWARD_COMPATIBLE 1
+client
+push-peer-info
+server-poll-timeout 5
+dev tun
+dev-type tun
+nobind
+persist-key
+auth-user-pass
```
Comment (haugene) [inline openvpn/ironsocket/Argentina.ovpn]: Needs to be updated to read credentials from file, like this: https://github.com/haugene/docker-transmission-openvpn/blob/master/openvpn/pia/AU%20Melbourne.ovpn#L13

**Comment by haugene** [comment]
Hey and thanks for creating this PR. Have you tested it locally?

I've added a comment in one of the ovpn files, you need to make it read the login credentials from a file like the other configs. Same goes for all the other files.

Doesn't look like you have to do more than that to make it work, all certificates seems to be inlined in the config files. Should also update README and add a default.ovpn symlink like the other providers.

**Comment by brossow** [comment]
@haugene I'll see what I can do about making those updates. Haven't tested and hoped getting it into the dev branch would at least prompt someone who knows what they're doing to tweak as needed. I haven't actually used it myself yet because I need the IronSocket stuff myself and decided to get the ball rolling since no one else had. I'll update when I have looked at the change you described. :)

**Comment by brossow** [comment]
@haugene Please take a look at my changes at your convenience and let me know if anything else should be changed. I'm very new to this but work well with examples. As mentioned in the commits, I did the following:

1. Updated 'auth-user-pass' to read login credentials from a file
2. Updated 'ca', 'cert', 'key' and 'tls-cert' to read from files to simplify future maintenance
3. Added default.ovpn pointing to United-Kingdom.ovpn
4. Updated README to include IronSocket in the list of supported providers

**Comment by haugene** [comment]
Hello again. The changes looks great. Merging to dev now. Please let me know if you get it working :)
Thanks for contributing!

**Comment by brossow** [comment]
@haugene Just wanted to let you know that I finally got around to installing this today from the dev branch. Transmission and the new Ironsocket VPN stuff I added are working great on my Synology (thanks to the tips on issue #41).

**Pull request merged**